rust/main/agents/relayer/src/server/message_retry.rs

```diff
@@ -44,7 +44,7 @@ async fn retry_message(
 
     tracing::debug!(uuid = uuid_string, "Sending message retry request");
 
-    let (transmitter, mut receiver) = mpsc::channel(ENDPOINT_MESSAGES_QUEUE_SIZE);
+    let (transmitter, mut receiver) = mpsc::channel(1);
```

Comment:
nit: add a comment mentioning that this channel is only created to service this single retry request so we're expecting a single response from each transmitter end, hence why we're using a size of 1